tsdb/head.go

```diff
@@ -231,6 +235,26 @@ func newHeadMetrics(h *Head, r prometheus.Registerer) *headMetrics {
 	return m
 }
 
+const cardinalityCacheExperationTime int64 = 30 //seconds
+
+//Return top 10 Highest Cardinality stats for Labels and Value Names
```

Comment:
Comments should have a space after the `//` and end in a full stop (needs to be updated in a few places).